src/yin_yang.py

```diff
@@ -73,6 +76,8 @@ def run(self):
                 except Exception as e:
                     logger.error('Error while changing the theme in plugin ' + pl.name)
```

Comment:
I think having a notification when the theme change failed is more important than having one when succesfully changes. The visual cue for the theme change should be the theme change itself.

Reply:
I actually think it is useful since somtimes the screen freezes or becomes unresponsive during the change. With the notification we can see what is happening. 

But I also agree that a notification on a failed change would be useful, but this might be out of scope for this PR.